Sources/ArgumentParser/Documentation.docc/ArgumentParser.md

```diff
@@ -90,3 +90,4 @@ and then either calls your `run()` method or exits with a useful message.
 
 - <doc:ManualParsing>
 - <doc:ExperimentalFeatures>
+- <doc:CommandsAndSubcommands>
```

Comment:
This article was originally in the "Essentials" section before I moved the link into `ParsableCommand`. Did you have a hard time finding it? I'd rather restore it to its original spot, under "Getting Started".

Reply:
I think there should be a page that acts as a table of contents listing all the important articles. Also I don't think an article link can't be in two places at the same time. If all articles can be found in the `ArgumentParser` directory, `CommandsAndSubcommands` will likely be ignored.

Reply:
I found `CustomizingCommandHelp` also in the `ParsableCommand`. I think it's hard to find an article that only appears in the documentation of a symbol. Because I usually read Quick Help by Option-clicking a symbol instead of searching this symbol in the global documentation. And all text introductions can be seen in Quick Help before the launch of docc. We only need to go deep if we need to understand the methods under it.

Reply:
Article links can be included in multiple places. I agree that we should have the articles all visible from the root-level page — let's add links to both of those to the "Essentials" section, as well.

Reply:
Done